src/pages/SubmissionDetails.js

```diff
@@ -193,15 +219,17 @@ export const SubmissionDetails = ({ submissionId }) => {
                 field: 'state',
                 headerRenderer: () => h(Sortable, { sort, field: 'state', onSort: setSort }, ['Status']),
                 cellRenderer: ({ rowIndex }) => {
+                  const getStatus = state(paginatedPreviousRuns[rowIndex].state)
```

Comment:
nitpicking / thinking out loud - maybe this variable should be called `status`? if you agree, I would suggest it. not a big deal, though.